packages/web/components/control/token-select-with-drawer.tsx

```diff
@@ -92,6 +96,14 @@ export const TokenSelectWithDrawer: FunctionComponent<{
 
     const canSelectTokens = tokens.length > 1;
 
+    const onSelect = (tokenDenom: string) => {
```

Comment:
We could use `useCallback` here to prevent unnecessary re-renders.

Reply:
Thanks for the review! 🚀 

I generally avoid `useCallback` unless it needs to be included on a dependency list within a `useCallback` or `useMemo`. This [article](https://kentcdodds.com/blog/usememo-and-usecallback) has a great explanation. Let me know what you think.